tests/tests_unit/test_api/test_functions.py

```diff
@@ -910,6 +907,17 @@ def test_list_schedules_with_function_id_and_function_external_id_raises(self, c
             == excinfo.value.args[0]
         )
 
+    @pytest.mark.usefixtures("mock_function_schedules_response_with_xid")
+    def test_create_schedules_with_function_external_id(self, cognite_client):
+        res = cognite_client.functions.schedules.create(
+            name="my-schedule",
+            function_external_id="my-func",
+            cron_expression="*/5 * * * *",
+            description="Hi",
+        )
+
+        assert isinstance(res, FunctionSchedule)
```

Comment:
I think you should also add a check here on the mock that `function_id` is what is passed to `_post` (and not external ID)

Reply:
Done. Not sure if this is the right way to test variables passed but it seems to test the necessary functionality